.github/workflows/_reusable_app_release.yml

```diff
@@ -117,6 +121,8 @@ jobs:
       C3_APP_CENTER_APP_NAME_PRODUCTION: ${{ secrets.C3_APP_CENTER_APP_NAME_PRODUCTION }}
       BETA_TESTFLIGHT_LINK: ${{ secrets.BETA_TESTFLIGHT_LINK }}
       PLAYGROUND_TESTFLIGHT_LINK: ${{ secrets.PLAYGROUND_TESTFLIGHT_LINK }}
+      COUNTLY_PRODUCTION_KEY: ${{ secrets.COUNTLY_PRODUCTION_KEY }}
+      COUNTLY_INTERNAL_KEY: ${{ secrets.COUNTLY_INTERNAL_KEY }}
```

Comment:
After removing the key from the other repo, should we regenerate it and update in the new environment just to be sure it has changed in case someone access the old repo?
https://github.com/wireapp/wire-ios-build-assets/commit/143da17b53df08c52b1dc4db95894d08cf5ec14e

Reply:
@markusfassbender you mean rotate the keys?

Reply:
yeah but as it's only one then old builds break. doesn't make sense, sorry